Add OSSci MOSS IG presentation and pavilion for the book (#240)
software-gardening/almanack: src/book/garden-circle/pavilion.md

# Pavilion

> The Software Gardening Almanack sometimes is presented through events which we want to share with you under this pavilion.

Reviewer: ```suggestion
We sometimes present the Software Gardening Almanack at events, which we want to share with you under this pavilion.
```

Author: Thanks, looks great!
